### ISSUE TITLE
Credo.Check.Refactor.CyclomaticComplexity error should be spelled out

### ISSUE THREAD

**Issue by bmitc** [rrrene/credo#951]
### Precheck

* Proposals for new features should be submitted via: https://github.com/rrrene/credo-proposals
* For bugs, please do a quick search and make sure the bug has not yet been reported here

### Environment

* Credo version (`mix credo -v`): `1.6.2-ref.main.bf1d5b8+uncommittedchanges`
* Erlang/Elixir version (`elixir -v`):
    ```
    Erlang/OTP 24 [erts-12.1.4] [source] [64-bit] [smp:16:16] [ds:16:16:10] [async-threads:1] [jit]

    Elixir 1.13.2 (compiled with Erlang/OTP 22)
    ```
* Operating system: Windows 11

### What were you trying to do?

I ran `mix.bat credo` and got the following message from Credo: `Function is too complex (CC is 23, max is 9).` The function it was reporting this on is:

```elixir
def eval(%Transaction{from: from, to: to, value: value} = t, contract) do
    case contract do
      x when is_number(x) -> x
      s when is_binary(s) -> s
      [] -> true
      {} -> true
      true -> true
      false -> false
      {:if, condition, tr, fa} -> if eval(t, condition), do: eval(t, tr), else: eval(t, fa)
      {:+, left, right} -> eval(t, left) + eval(t, right)
      {:*, left, right} -> eval(t, left) * eval(t, right)
      {:-, left, right} -> eval(t, left) - eval(t, right)
      {:==, left, right} -> eval(t, left) == eval(t, right)
      {:>, left, right} -> eval(t, left) > eval(t, right)
      {:<, left, right} -> eval(t, left) < eval(t, right)
      {:and, left, right} -> eval(t, left) and eval(t, right)
      {:or, left, right} -> eval(t, left) or eval(t, right)
      :from -> from
      :to -> to
      :value -> value
      _ -> false
    end
  end
```

### Expected outcome

I expected two things:
1. Credo should tell me what check generates a given message. This is probably a feature request.
2. The message should spell out the `CC` abbreviation.

### Actual outcome

In this case, I had little idea what was going on at first. I didn't know what `CC` meant, and since this error was reported on the above function, I initially assumed it meant "case clauses". I actually didn't count the clauses (which is 19), but I knew that 23 was close enough that that seemed to match. But then I had a hard time understanding which Credo check in `.credo.exs` was generating this so that I could configure or disable it. I only found out what `CC` meant by searching Google and GitHub for the Credo message. Through that search I found the module and code and was able to see what it meant.

https://github.com/rrrene/credo/blob/3764952cbcc02c19b053f801051907aa47751c5a/lib/credo/check/refactor/cyclomatic_complexity.ex#L171

**Comment by rrrene** [comment]
You can display the check which is producing an issue by using `--verbose`.

I updated the message based on your suggestion :+1:
